### ISSUE TITLE
Hi ,eBay dose use it in a production envirement ? 

### ISSUE THREAD

**Issue by triump2020** [eBay/Jungle#117]

**Comment by greensky00** [comment]
@triump2020 Yes, it has been used in various use cases in many servers as a storage engine.

**Comment by triump2020** [comment]
@greensky00   Thanks for your quick response !    

    In ebay,    Has there been any case in where  jungle has been used  as a wal-raft-log store ?  we want to multiplex it, namely to improve the performance for the disk io ,  we plan to put multi wal-logs come from different raft groups into one Jungle instance which  correspond to one disk.   is it feasible ?    how  did it performs?    Thanks again !

**Comment by triump2020** [comment]
@greensky00  thanks for your quick response ! 
      BTW,  in ebay,  has there been any case in where  jungle has been used  as a wal-raft-log store ?  we want to multiplex it, 
namely to improve the performance for the disk io ,  we plan to put multi wal-logs come from different raft groups into one 
Jungle instance which  correspond to one disk.   is it feasible ?    how  did it performs?    Thanks again !

**Comment by greensky00** [comment]
@triump2020 
Yes, one of the use cases of Jungle is the Raft log store, but we use one Jungle instance for each Raft group (there can be multiple isolated instances in a single process). If you want to share multiple log stores on a single Jungle instance, you may need to have a mapping from each log store's index number to Jungle's sequence number. That won't be easy while the benefit is questionable compared to having multiple Jungle instances.

**Comment by triump2020** [comment]
@greensky00   Got  it ,  thanks!
